Examples/demodemo.lisp

```diff
@@ -165,16 +165,18 @@ argument to avoid creating too many functions with similar name."))
            ) )))
 
 (defun make-test-label (ax ay)
-  (labelling (:label "Some Label"
-                     :align-x ax
-                     :label-alignment ay
-                     :foreground +WHITE+
-                     :background +PALETURQUOISE4+
-                     :text-style (make-text-style :sans-serif :roman :normal))
+  ;; Please keep the silly "good" so the label text goes below the
+  ;; baseline.
+  (labelling (:label "Some good Label"
```

Comment:
Please add initials and date of the comment, i.e `... - jm 2019-12-14`

Reply:
Done.